--- src/msw/spinctrl.cpp.old	2014-11-09 19:23:38.236202674 +0100
+++ src/msw/spinctrl.cpp	2014-11-09 19:25:04.151205738 +0100
@@ -334,6 +334,10 @@
     WXDWORD exStyle = 0;
     WXDWORD msStyle = MSWGetStyle(GetWindowStyle(), & exStyle) ;
 
+    // Scroll text automatically if there is not enough space to show all of
+    // it, this is better than not allowing to enter more digits at all.
+    msStyle |= ES_AUTOHSCROLL;
+
     // calculate the sizes: the size given is the toal size for both controls
     // and we need to fit them both in the given width (height is the same)
     wxSize sizeText(size), sizeBtn(size);
