From 0df6bfa5a0dbebf8466284f47d17ecc6ff8a2b01 Mon Sep 17 00:00:00 2001
From: Martin Koegler <martin.koegler@chello.at>
Date: Thu, 14 May 2015 13:48:11 +0200
Subject: [PATCH] Fix #16984

---
 include/wx/filefn.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/wx/filefn.h b/include/wx/filefn.h
index 4815807..edfd8d0 100644
--- a/include/wx/filefn.h
+++ b/include/wx/filefn.h
@@ -376,7 +376,7 @@ enum wxPosixPermissions
 
     // finally the default char-type versions
     #if wxUSE_UNICODE
-        #if wxUSE_UNICODE_MSLU || defined(__WX_STRICT_ANSI_GCC__)
+        #if wxUSE_UNICODE_MSLU
             // implement the missing file functions in Win9x ourselves
             WXDLLIMPEXP_BASE int wxMSLU__wopen(const wxChar *name,
                                                int flags, int mode);
-- 
2.1.4

