diff --git a/src/grandorgue/CMakeLists.txt b/src/grandorgue/CMakeLists.txt
--- a/src/grandorgue/CMakeLists.txt
+++ b/src/grandorgue/CMakeLists.txt
@@ -28,10 +28,10 @@ if (NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
    add_option(-funroll-loops)
    add_definitions(-DNDEBUG)
 endif ()
-add_option(-mmmx)
-add_option(-msse)
-add_option(-msse2)
-add_option(-msse3)
+#add_option(-mmmx)
+#add_option(-msse)
+#add_option(-msse2)
+#add_option(-msse3)
 
 add_option(-mstackrealign)
 
