diff --git a/src/grandorgue/CMakeLists.txt b/src/grandorgue/CMakeLists.txt
--- a/src/grandorgue/CMakeLists.txt
+++ b/src/grandorgue/CMakeLists.txt
@@ -21,10 +21,10 @@ include(${CMAKE_SOURCE_DIR}/cmake/AddLinkerOption.cmake)
 include(${CMAKE_SOURCE_DIR}/cmake/CopyWxTranslations.cmake)
 include(${CMAKE_SOURCE_DIR}/cmake/CopyDependencies.cmake)
 
-add_option(-mmmx)
-add_option(-msse)
-add_option(-msse2)
-add_option(-msse3)
+#add_option(-mmmx)
+#add_option(-msse)
+#add_option(-msse2)
+#add_option(-msse3)
 
 add_option(-mstackrealign)
 
