diff --git a/CMakeLists.txt b/CMakeLists.txt
index a6427bf..8c4f8ad 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -102,10 +102,10 @@ elseif (CMAKE_BUILD_TYPE STREQUAL "Release")
    add_option(-funroll-loops)
    add_definitions(-DNDEBUG)
 endif ()
-add_option(-mmmx)
-add_option(-msse)
-add_option(-msse2)
-add_option(-msse3)
+#add_option(-mmmx)
+#add_option(-msse)
+#add_option(-msse2)
+#add_option(-msse3)
 
 add_option(-mstackrealign)
 
